src/main/java/sirius/db/mixing/DateRange.java

```diff
@@ -248,7 +266,7 @@ public String getKey() {
      * @return the start of the date range
      */
     public LocalDateTime getFrom() {
-        return from;
+        return fromSupplier != null ? fromSupplier.get() : null;
```

Comment:
What situation could actually bring such a supplier to be null?

Reply:
If the constructor would be called without a from supplier :)